src/mutation-observer.js

```diff
@@ -100,13 +100,13 @@ MutationObserver.fixIe = function () {
   }
 
   // We have to call the old innerHTML getter and setter.
-  var oldInnerHtml = Object.getOwnPropertyDescriptor(elementPrototype, 'innerHTML');
```

Comment:
Nitpick but I do see your reasoning. All uppercase is a shitty convention that should be done away with. You can't tell where one word ends and another begins as quickly: SomeHTMLString vs SomeHtmlString. No need to change, but for future reference that's a convention we follow.